src/main/scala/mesosphere/marathon/core/instance/update/InstanceUpdateOpResolver.scala

```diff
@@ -31,11 +34,31 @@ private[marathon] class InstanceUpdateOpResolver(
     */
   def resolve(op: InstanceUpdateOperation)(implicit ec: ExecutionContext): Future[InstanceUpdateEffect] = {
     op match {
+      case op: Schedule =>
+        // TODO(karsten): Create events
```

Comment:
these TODOs need to be resolved before we merge this for release, right? Should we tag them with a JIRA issue?

Reply:
https://jira.mesosphere.com/browse/MARATHON-8350